oauth2_lib/fastapi.py

```diff
@@ -251,9 +252,10 @@ async def introspect_token(self, async_request: AsyncClient, token: str) -> OIDC
         await self.check_openid_config(async_request)
         assert self.openid_config
 
+        endpoint = self.openid_config.introspect_endpoint or self.openid_config.introspection_endpoint
```

Comment:
oidc-proxy does not like token in the body instead of query, so will implement a check with which property is used